### ISSUE TITLE
broken variable after a linter pass

### ISSUE THREAD

**Issue by pums974** [cphyc/fortran-syntax#6]
Hi,

Thank you for this linter, it will be very usefull for us

However, I think I have spotted a problem with your linter.

before
```fortran
module my_module
contains
     subroutine my_subroutine()
          implicit none
          integer :: bug
          bug= 1
     end subroutine my_subroutine
end module my_module
```
after `fortran-linter --linelength 100 -i file.f90`
```fortran
module my_module
contains
     subroutine my_subroutine()
          implicit none
          integer :: bug
          bu = g1
     end subroutine my_subroutine
end module my_module
```

I'm using the latest version available on PyPy and my Python version is 3.9.2

**Comment by cphyc** [comment]
Should be fixed as of 25e74d208a0b198008d1976a16c3215fcd5910ff and v1.0.7 (available on pip). Re-open if this is not the case!

**Comment by pums974** [comment]
Yes, fixed.

Thank you, and thanks for your reactivity
